scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/ram/RamCompute.java

```diff
@@ -0,0 +1,55 @@
+package org.ow2.proactive.scheduler.policy.ram;
+
+import java.io.Serializable;
+import java.lang.management.ManagementFactory;
+
+import org.objectweb.proactive.annotation.PublicAPI;
+import org.objectweb.proactive.core.runtime.LocalNode;
+import org.objectweb.proactive.core.runtime.ProActiveRuntimeImpl;
+
+import com.sun.management.OperatingSystemMXBean;
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Replace this usage of Sun classes by ones from the Java API. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1191)